src/script/components/list/ConversationListCell.tsx

```diff
@@ -123,153 +111,144 @@ const ConversationListCell: React.FC<ConversationListCellProps> = ({
   };
 
   return (
-    <div
-      ref={setViewportElementRef}
-      data-uie-name={dataUieName}
-      data-uie-uid={conversation.id}
-      data-uie-value={displayName}
-      role="button"
-      tabIndex={0}
-      className={cx('conversation-list-cell', {'conversation-list-cell-active': isSelected})}
-      onClick={onClick}
-      onKeyPress={handleKeyPress}
-    >
-      {isInViewport && (
-        <>
-          <div
-            className={cx('conversation-list-cell-left', {
-              'conversation-list-cell-left-opaque': removedFromConversation || users.length === 0,
-            })}
-          >
-            {isGroup && <GroupAvatar className="conversation-list-cell-avatar-arrow" users={users} />}
-            {!isGroup && !!users.length && (
-              <div className="avatar-halo">
-                <Avatar participant={users[0]} avatarSize={AVATAR_SIZE.SMALL} />
-              </div>
-            )}
-          </div>
-          <div className="conversation-list-cell-center">
-            {is1to1 && selfUser.inTeam() ? (
-              <AvailabilityState
-                className="conversation-list-cell-availability"
-                availability={availabilityOfUser}
-                label={displayName}
-                theme={isSelected}
-                dataUieName="status-availability-item"
-              />
-            ) : (
-              <span className={cx('conversation-list-cell-name', {'accent-text': isSelected})}>{displayName}</span>
-            )}
-            <span className="conversation-list-cell-description" data-uie-name="secondary-line">
-              {cellState.description}
-            </span>
-          </div>
-          <div className="conversation-list-cell-right">
+    <li ref={setViewportElementRef}>
+      <div
```

Comment:
if this is role button maybe it should actually be a button element

Reply:
This one is correct as it has a div inside's do should stay like it is